plugins/available/gitstatus.plugin.bash

```diff
@@ -21,4 +21,6 @@ if [[ -d ${SCM_GIT_GITSTATUS_DIR} ]]; then
   # Start the actual gitstatus binary
   gitstatus_stop && gitstatus_start -s -1 -u -1 -c -1 -d -1
   export SCM_GIT_USE_GITSTATUS=true
+else
+	_log_warning "Could not find gitstatus directory in ${SCM_GIT_GITSTATUS_DIR}. Please specify directory location using SCM_GIT_GITSTATUS_DIR."
```

Comment:
Thinking these types of messages should also add some type of standard `"You can disable the plugin ..."` messaging and / or "See bashit help ..." messaging

Reply:
Not sure how can we standardize that. Some logs should display that but some logs should not...
We can add this manually to each message that needs it